services/network/__main__.py

```
@@ -0,0 +1,144 @@
+            "timestamp": datetime.now(timezone.utc).isoformat(),
+        })
+
+    app.register_blueprint(create_blueprint())
+    return app
+
+
+def _run_api_server(app: Flask) -> None:
+    """Run Flask API server in a background thread."""
+    log = logging.getLogger(__name__)
+    try:
+        app.run(host="0.0.0.0", port=PORT, debug=False, use_reloader=False)
```

Ruff (0.15.13)

[error] Possible binding to all interfaces

(S104)